explore/src/main/scala/explore/config/ImagingModesTable.scala

```diff
@@ -42,10 +51,20 @@ import lucuma.ui.table.ColumnSize.*
 import lucuma.ui.table.hooks.*
 
 final case class ImagingModesTable(
-  userId: Option[User.Id],
-  matrix: ImagingModesMatrix,
-  units:  WavelengthUnits
-) extends ReactFnProps(ImagingModesTable.component)
+  userId:              Option[User.Id],
+  imaging:             ImagingConfigurationOptions,
+  matrix:              ImagingModesMatrix,
+  constraints:         ConstraintSet,
+  targets:             List[ItcTarget],
+  baseCoordinates:     Option[CoordinatesAtVizTime],
+  customSedTimestamps: List[Timestamp],
+  units:               WavelengthUnits
```

Comment:
Do we show wavelengths here? Maybe for the filters?

Reply:
Yes, there are `λ` and a `Δλ` columns that come from the filter.